CONTRIBUTING.md

```diff
@@ -61,14 +61,15 @@ in your development environment.
 ## Getting Started
 ### Bootstrapping
 
-The project is managed as a [monorepo] using [lerna].
+The project is managed as a [monorepo] using [yarn] and [turborepo].
 
 [monorepo]: https://github.com/babel/babel/blob/main/doc/design/monorepo.md
-[lerna]: https://github.com/lerna/lerna
+[yarn]: https://classic.yarnpkg.com/en/
+[turborepo]: https://turborepo.org
 
 1. Check out this respository and change directory to its root.
-2. Run `yarn install && yarn build` to install lerna, bootstrap the repository
-   and perform an initial build and test cycle.
+2. Run `yarn install && yarn test` to install the necessary dependencies,
```

Comment:
Why yarn test?

Reply:
`test` implies `build` in the `turbo` pipeline, and it makes sure your environment is healthy (has all requirements, etc...)